apps/prairielearn/elements/pl-order-blocks/pl-order-blocks.js

```diff
@@ -3,11 +3,212 @@
+              }
+            } else {
+              const blockIndex = optionsBlocks.indexOf(block);
+              if (blockIndex > 0) {
+                optionsBlocks[blockIndex - 1].focus();
+              }
+            }
+          }
+        };
+        switch (ev.key) {
+          case 'Enter':
+            handleKey(ev, block, () => block.classList.add('pl-order-blocks-selected'));
+            break;
+          case 'ArrowUp':
+            handleKey(
+              ev,
+              block,
+              () => {
+                moveWithinOptionsOrDropzone(false);
+              },
+              false,
+            );
+            break;
+          case 'ArrowDown':
+            handleKey(
+              ev,
+              block,
+              () => {
+                moveWithinOptionsOrDropzone(true);
+              },
+              false,
+            );
+            break;
+          case 'ArrowLeft':
+            handleKey(
+              ev,
+              block,
+              () => {
+                moveBetweenOptionsOrDropzone(true);
+              },
+              false,
+            );
+            break;
+          case 'ArrowRight':
+            handleKey(
+              ev,
+              block,
+              () => {
+                moveBetweenOptionsOrDropzone(false);
+              },
+              false,
+            );
+            break;
+        }
+      } else {
+        switch (ev.key) {
+          case 'ArrowDown':
+            handleKey(ev, block, () => {
+              if (block.nextElementSibling) {
+                block.nextElementSibling.insertAdjacentElement('afterend', block);
+              }
+            });
+            break;
+          case 'ArrowUp':
+            handleKey(ev, block, () => {
+              if (block.previousElementSibling) {
+                block.previousElementSibling.insertAdjacentElement('beforebegin', block);
+              }
+            });
+            break;
+          case 'ArrowLeft':
+            handleKey(ev, block, () => {
+              if (inDropzone(block)) {
+                const currentIndent = getIndentation(block);
+                if (currentIndent > 0) {
+                  setIndentation(block, getIndentation(block) - 1);
+                } else {
+                  $(optionsElementId)[0].insertAdjacentElement('beforeend', block);
+                  correctPairing(block);
+                }
+              }
+            });
+            break;
+          case 'ArrowRight':
+            handleKey(ev, block, () => {
+              if (!inDropzone(block)) {
+                $(dropzoneElementId)[0].insertAdjacentElement('beforeend', block);
+              } else if (enableIndentation) {
+                setIndentation(block, getIndentation(block) + 1);
+              }
+            });
+            break;
+          case 'Escape':
+            handleKey(ev, block, removeSelectedAttribute);
+            break;
```

Comment:
My experience is that it would be very useful if `Enter` also performs this action. In other words: Enter to select, also Enter to deselect. What do you think? Won't block on that, though. The thought is: Enter to select the block, arrows to position it where I want, Enter to settle on the location.

Reply:
At this point I would prefer to keep escape for deselect. My worry about enter for select and deselect is that you can forget what state you are in, selected or deselected. We are planning on doing some user studies though so we can keep this in mind as a possible thing to A/B test on

Happy to add space to select also.